.github/workflows/main.yml

```diff
@@ -23,6 +23,9 @@ jobs:
       run: |
         dotnet tool install --version 5.0.142902 --tool-path ./ dotnet-format --add-source https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-tools/nuget/v3/index.json
         ./dotnet-format . --check -v diagnostic
+    - name: Publish
+      run: |
+        dotnet publish neo-modules.sln
```

Comment:
I think it is not necessary. We have used `dotnet test` already. It will fail if build fails.

Reply:
Does it test all src or just those linked to the test?